test/stdlib/tgmath.swift

```diff
@@ -174,9 +173,9 @@ MathTests.test("Other functions") {
   expectEqual(lgamma(gx).1, 1)
   expectEqualWithAccuracy(lgamma(dx).0, lgamma(fx).0, lgamma(gx).0, 2.2527126517342055, ulps: 1)
 
-  expectEqual(remquo(dz, dy).1, 1)
-  expectEqual(remquo(fz, fy).1, 1)
-  expectEqual(remquo(gz, gy).1, 1)
+  expectEqual(remquo(dz, dy).1 & 7, 1)
```

Comment:
Please add a comment explaining where the `& 7` comes from. It can be just the sentence in my review note.